string obj_name
float32 prob
int32 cluster_idx
